Feat(*): 모니터링을 위해 개발 환경에서 파일로 로깅

morenow98 opened the pull request:
## PR 변경된 내용
- 로컬 환경에서는 기본 콘솔 로그 설정 사용
- 개발 환경에서는 logback-dev.xml 파일로 설정
  - 일반 로그와 sql 로그를 각각 다른 파일에 로깅하도록 설정
  - Hibernate SQL 로그를 별도 파일에 기록하도록 설정하고, Hibernate의 나머지 로그는 OFF로 조정해 저장되지 않도록 설정
  - show_sql=true 설정은 콘솔 출력을 하는 설정이므로 개발 환경에서는 false로 설정
### 컨테이너 환경에서 elk 지침 및 자동화
- 로그 경로를 환경변수로 처리. 환경변수 최신화 필요
- CD 과정 중에 로그 관련 볼륨 (logsVolume) 추가
- script 폴더에 elk 관련 설정과 docker compose 파일 추가

## 추가 내용
- 로컬에서 jwt 유효 기간이 기본값과 동일하게 중복 삽입되어 삭제

## 참조
Closes #49

morenow98 committed:
Refactor(*): 잘못된 주석 및 중복 표현 삭제

morenow98 committed:
Feat(*): 개발환경에 따라 다른 로깅 전략 구현

- 로컬 환경에서는 기본 콘솔 로그 설정 사용
- 개발 환경에서는 일반 로그와 sql 로그를 각각 다른 파일에 로깅하도록 설정

InHyeok-J commented on jabiseo-api/src/main/resources/logback-dev.xml:
```diff
@@ -0,0 +1,42 @@
+    <!-- 일반 로그 파일 설정 -->
+    <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>/Users/morenow/Desktop/jabiseo/logs/jabiseo.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>/Users/morenow/Desktop/jabiseo/logs/jabiseo.%d{yyyy-MM-dd}.log</fileNamePattern>
+            <maxHistory>30</maxHistory>
```
경로가 하드코딩 돼있는데 이 부분을 다른 방식 
ex) 상대 경로/ 환경변수 등으로 변경해야 할듯?

InHyeok-J replied:
추가로 컨테이너 환경에서는 어떻게 설정할지 
파일 경로를 지정하고 볼륨 마운트를 할건지.. 등등 이런거가 PR 설명에 없는거보면 고려가 됐는지 궁금함!

morenow98 replied:
수정완료!

morenow98 committed:
Feat(*): 로그 경로를 환경변수로 처리

morenow98 committed:
Feat(*): 컨테이너 환경에서 elk 관련 및 자동화 설정

- CD 과정 중에 로그 관련 볼륨 (logsVolume) 추가
- script 폴더에 elk 관련 설정과 docker compose 파일 추가

InHyeok-J reviewed (approved):
굳굳고생했어~개발서버에서 테스트 고고
